publish-cli/pom.xml

```diff
@@ -125,15 +125,9 @@
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.powermock</groupId>
-            <artifactId>powermock-module-junit4</artifactId>
-            <version>${powermock.version}</version>
-            <scope>test</scope>
-        </dependency>
-        <dependency>
-            <groupId>org.powermock</groupId>
-            <artifactId>powermock-api-mockito2</artifactId>
-            <version>${powermock.version}</version>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-inline</artifactId>
+            <version>${mockito.version}</version>
```

Comment:
With mockito 5 inline mockito is supported by default additional configuration for mockmacker inline is not needed if used

Reply:
Uplifted to Mockito 5.3.1. Removed mockito-inline.